experimental/generation/generator/packages/library/templates/generator.lg

```diff
@@ -308,7 +311,28 @@ ${'#'} ${property}Entity_getValueText(val)
             ]
         }
     ]
-    ${designerMissing(property)}     
+    ${designerMissing(property)
+    }
+}
+```
+
+# helpDialog
+- ```
+{
+    "$schema": "${appSchema}",
+    "$kind": "Microsoft.OnEndOfActions",
+    "condition": "$retries > 0 && $lastIntent != 'Help'",
+    "runOnce": true,
+    "priority": "=lib.missingPriority('${property}')",
```

Comment:
I suspect this should be a priority of 0, i.e. it should run before anything else.

Reply:
I change priority to 0, but in that case, the help info will been shown many times if the wrong input is fed.